### ISSUE TITLE
Return actual error message when action upload_to_testflight runs on error

### ISSUE THREAD

**Issue by JohanBeumer** [fastlane/fastlane#17181]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
I want to be able to get the error I get from testflight, I'm uploading an .ipa that was already uploaded before, in my fastfile so I can pass it on to slack. The console shows the error :

`ERROR ITMS-90189: "Redundant Binary Upload. You've already uploaded a build with build number '1' for version number '1.0.0'. Make sure you increment the build string before you upload your app to App Store Connect. Learn more in Xcode Help (http://help.apple.com/xcode/mac/current/#/devba7f53ad4)."
Return status of iTunes Transporter was 1: ERROR ITMS-90189: "Redundant Binary Upload. You've already uploaded a build with build number '1' for version number '1.0.0'. Make sure you increment the build string before you upload your app to App Store Connect. Learn more in Xcode Help (http://help.apple.com/xcode/mac/current/#/devba7f53ad4)."
The call to the iTMSTransporter completed with a non-zero exit status: 1. This indicates a failure.`

In my fastfile I'm catching the message like this : 

```
begin
upload_to_testflight()
rescue => exception
    puts exception.to_s
end
```

This shows the text :
`Error uploading ipa file, for more information see above ^`

Instead of this message I would like to see the message that was given in the console. Especially in a cicd-scenario that's the information I want to pass to the slack channel.

The message in the fastfile comes from line 37 of `pilot/lib/pilot/build_manager.rb` (2.158.0)
The actual error message is available in `fastlane_core/lib/fastlane_core/itunes_transporters.rb`. There it is added to `@errors` at line 62 and added to `@errors` (again) at line 67. I would love to see `@errors` also being returned at line 96, so it can be used in `build_manager.rb` and is available in the fastfile to send to slack. Also put it in `@errors` only once.

##### Command executed
bundle exec fastlane ios upload_to_testflight

##### Complete output when running fastlane, including the stack trace and command used
23:22:17]: ----------------------------------
[23:22:17]: --- Step: upload_to_testflight ---
[23:22:17]: ----------------------------------
[23:22:17]: Login to App Store Connect
Could not find team with name '', trying to fallback to default team
[23:22:18]: Login successful
[23:22:20]: Ready to upload new build to TestFlight (App:)...
[23:22:20]: Fetching password for transporter from environment variable named `FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD`
[23:22:20]: Going to upload updated app to App Store Connect
[23:22:20]: This might take a few minutes. Please don't interrupt the script.
[23:22:35]: [Transporter Error Output]: ERROR ITMS-90189: "Redundant Binary Upload. You've already uploaded a build with build number '1' for version number '1.0.0'. Make sure you increment the build string before you upload your app to App Store Connect. Learn more in Xcode Help (http://help.apple.com/xcode/mac/current/#/devba7f53ad4)."
[23:22:35]: Transporter transfer failed.
[23:22:35]: 
[23:22:35]: ERROR ITMS-90189: "Redundant Binary Upload. You've already uploaded a build with build number '1' for version number '1.0.0'. Make sure you increment the build string before you upload your app to App Store Connect. Learn more in Xcode Help (http://help.apple.com/xcode/mac/current/#/devba7f53ad4)."
[23:22:35]: ["ERROR ITMS-90189: \"Redundant Binary Upload. You've already uploaded a build with build number '1' for version number '1.0.0'. Make sure you increment the build string before you upload your app to App Store Connect. Learn more in Xcode Help (http://help.apple.com/xcode/mac/current/#/devba7f53ad4).\"\r", "Return status of iTunes Transporter was 1: ERROR ITMS-90189: \"Redundant Binary Upload. You've already uploaded a build with build number '1' for version number '1.0.0'. Make sure you increment the build string before you upload your app to App Store Connect. Learn more in Xcode Help (http://help.apple.com/xcode/mac/current/#/devba7f53ad4).\"\r"]
[23:22:35]: ["ERROR ITMS-90189: \"Redundant Binary Upload. You've already uploaded a build with build number '1' for version number '1.0.0'. Make sure you increment the build string before you upload your app to App Store Connect. Learn more in Xcode Help (http://help.apple.com/xcode/mac/current/#/devba7f53ad4).\"\r", "Return status of iTunes Transporter was 1: ERROR ITMS-90189: \"Redundant Binary Upload. You've already uploaded a build with build number '1' for version number '1.0.0'. Make sure you increment the build string before you upload your app to App Store Connect. Learn more in Xcode Help (http://help.apple.com/xcode/mac/current/#/devba7f53ad4).\"\r", "The call to the iTMSTransporter completed with a non-zero exit status: 1. This indicates a failure."]
[23:22:35]: [iTMSTransporter] [2020-09-07 23:22:35 CEST] <main> DBG-X:   parameter ErrorMessage = ERROR ITMS-90189: "Redundant Binary Upload. You've already uploaded a build with build number '1' for version number '1.0.0'. Make sure you increment the build string before you upload your app to App Store Connect. Learn more in Xcode Help (http://help.apple.com/xcode/mac/current/#/devba7f53ad4).

[23:22:35]: [iTMSTransporter]  (1102)

[23:22:35]: [iTMSTransporter] [2020-09-07 23:22:35 CEST] <main> DBG-X:   parameter ShouldUseRESTAPIs = false

[23:22:35]: [iTMSTransporter] [2020-09-07 23:22:35 CEST] <main> ERROR: ERROR ITMS-90189: "Redundant Binary Upload. You've already uploaded a build with build number '1' for version number '1.0.0'. Make sure you increment the build string before you upload your app to App Store Connect. Learn more in Xcode Help (http://help.apple.com/xcode/mac/current/#/devba7f53ad4)."

[23:22:35]: [iTMSTransporter] [2020-09-07 23:22:35 CEST] <main> DBG-X: The error code is: 1102

[23:22:35]: [iTMSTransporter] [2020-09-07 23:22:35 CEST] <main>  INFO: Done performing authentication.

[23:22:35]: [iTMSTransporter] 

[23:22:35]: [iTMSTransporter] 

[23:22:35]: [iTMSTransporter] 

[23:22:35]: [iTMSTransporter] Package Summary:

[23:22:35]: [iTMSTransporter]  

[23:22:35]: [iTMSTransporter] 1 package(s) were not uploaded because they had problems:

[23:22:35]: [iTMSTransporter] 	/var/folders/q3/yc_hmzw141s0nd8901mws81c0000gn/T/d20200907-37034-pxjmpo/1194496471.itmsp - Error Messages:

[23:22:35]: [iTMSTransporter] 		ERROR ITMS-90189: "Redundant Binary Upload. You've already uploaded a build with build number '1' for version number '1.0.0'. Make sure you increment the build string before you upload your app to App Store Connect. Learn more in Xcode Help (http://help.apple.com/xcode/mac/current/#/devba7f53ad4)."

[23:22:35]: [iTMSTransporter] [2020-09-07 23:22:35 CEST] <main> DBG-X: Returning 1

[23:22:35]: iTunes Transporter output above ^
[23:22:35]: ERROR ITMS-90189: "Redundant Binary Upload. You've already uploaded a build with build number '1' for version number '1.0.0'. Make sure you increment the build string before you upload your app to App Store Connect. Learn more in Xcode Help (http://help.apple.com/xcode/mac/current/#/devba7f53ad4)."
Return status of iTunes Transporter was 1: ERROR ITMS-90189: "Redundant Binary Upload. You've already uploaded a build with build number '1' for version number '1.0.0'. Make sure you increment the build string before you upload your app to App Store Connect. Learn more in Xcode Help (http://help.apple.com/xcode/mac/current/#/devba7f53ad4)."
The call to the iTMSTransporter completed with a non-zero exit status: 1. This indicates a failure.
[23:22:35]: Error uploading ipa file, for more information see above
